scanomatic/image_analysis/image_grayscale.py

```diff
@@ -437,9 +448,10 @@ def detect_grayscale(im_trimmed, grayscale: Grayscale):
             ),
         )
 
-    if len(grayscale_segment_centers) == 0:
+    if grayscale_segment_centers is None:
```

Comment:
they can't have length 0 still?

Reply:
I guess that could be possible still. Added it again